package.json

```diff
@@ -2,14 +2,16 @@
 	"name": "prismjs",
 	"version": "1.29.0",
 	"description": "Lightweight, robust, elegant syntax highlighting. A spin-off project from Dabblet.",
-	"main": "prism.js",
+  "type": "module",
+	"main": "dist/core/prism.js",
```

Comment:
Opt into the modern Node ESM standard, and point to the main file to get Prism from (there was no more `./prism.js`).